### ISSUE TITLE
Antivirus (F-Secure) issued positive alert of Volume2

### ISSUE THREAD

**Issue by InvertJuha** [irzyxa/Volume2#89]
Hello,

Antivirus we are using found Volume2 as infected software.

Reason:
Heuristic.HEUR/AGEN.1124176|Blocked|Memory|$StringUtils.sanitizeUrl($!alert.infectionObjectName)|

Additional information:
https://www.f-secure.com/v-descs/heuristic.shtml

Any ideas how to avoid these? What part of code could cause this? Is this false positive?

**Comment by irzyxa** [comment]
Yes, some antiviruses can detect virus. I think it is possible because of hooks. Volume2 uses hooks for keyboard and mouse to have possibility processing user actions for control volume.
